File: Python/158A-Next Round.py
n, k = map(int, input().split(" "))
rlt = 0
arr = list(map(int, input().split(" ")))
set = arr[k-1];

for i in range(n):
    if arr[i] >= arr[k-1] and arr[i] > 0: 
        rlt += 1
print(rlt)

File: C/158A-Next Round.c
#include <stdio.h>

int main () {
    int n, k, i, set, rlt = 0, arr[50] = {0, };
    scanf("%d %d", &n, &k);
    for(i = 0; i < n; i++){
        scanf("%d", &arr[i]);
    }

    for(i = 0; i < n; i++){
        if(arr[i] >= arr[k-1] && arr[i] > 0){
            rlt ++;
        }
    }
    printf("%d", rlt);
    return 0;
}
